tests/config/config.d/zookeeper_fault_injection.xml

```diff
@@ -1,6 +1,7 @@
 <clickhouse>
     <zookeeper>
         <use_compression>1</use_compression>
+        <use_xid_64>1</use_xid_64>
```

Comment:
Don't we want to randomize it somehow? I recently added setting randomization https://github.com/ClickHouse/ClickHouse/pull/69328, but it's for user profile setting, not sure if can be easily extended for other config sections, though

Reply:
I need to push a change with randomization added for stress tests.
For integration tests, I'm not yet sure if we can enable it all because older binaries are used in some tests.